src/server/protocol.ts

```diff
@@ -2010,6 +2011,19 @@ namespace ts.server.protocol {
         body?: Diagnostic[] | DiagnosticWithLinePosition[];
     }
 
+    export interface InfoDiagnosticsSyncRequest extends FileRequest {
+        command: CommandTypes.InfoDiagnosticsSync;
+        arguments: InfoDiagnosticsSyncRequestArgs;
+    }
+
+    export interface InfoDiagnosticsSyncRequestArgs extends FileRequestArgs {

```

Comment:
Can/should interface declarations be shared across the various kinds of diagnostics?